client/src/app/core/ui-services/user-media-perm.service.ts

```diff
@@ -0,0 +1,71 @@
+import { Injectable } from '@angular/core';
+
+import { TranslateService } from '@ngx-translate/core';
+
+import { OverlayService } from './overlay.service';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class UserMediaPermService {
+    public constructor(private translate: TranslateService, private overlayService: OverlayService) {}
+
+    public async requestMediaAccess(): Promise<void> {
+        const hasMicAccess: PermissionStatus | null = await this.detectPermStatus();
+        if (!hasMicAccess || hasMicAccess.state === 'prompt') {
+            await this.requestMicAccess();
+        } else if (hasMicAccess.state === 'denied') {
+            this.throwMicPermError();
+        }
+    }
+
+    /**
+     * `navigator.permissions.query` does only work in chrome
+     * This function detects if this method works at all.
+     * If it does not work, we try to access the media anyways without
+     * detecting the set permission beforehand.
+     * The negative result would be, that the user sees the
+     * overlay for a very short time.
+     * This cannot be avoided, but rather solves itself if more browsers
+     * start to support the given media query
+     */
```

Comment:
@emanuelschuetze to prevent questions why this behaves the way it dies, understanding this is important.